packages/core/src/helpers/chain.ts

```diff
@@ -79,6 +81,8 @@ export function getExplorerTransactionLink(transactionHash: string, chainId: Cha
       return `https://ftmscan.com/tx/${transactionHash}`
     case ChainId.Avalanche:
       return `https://snowtrace.io/tx/${transactionHash}`
+    case ChainId.MoonbaseAlpha:
+      return `https://moonbase.moonscan.io/tx/${transactionHash}`
```

Comment:
Is it multicall contract?
Can't see code:
https://moonbase.moonscan.io/address/0x4E2cfca20580747AdBA58cd677A998f8B261Fc21#code

Reply:
@wagmiwiz Would you verify it?

Reply:
@tt-marek I've verified the contract :) - https://moonbase.moonscan.io/address/0x4E2cfca20580747AdBA58cd677A998f8B261Fc21#code

Reply:
❤️